device/mellanox/x86_64-nvidia_sn5640-r0/Mellanox-SN5640-C448O16/port_config.ini

```diff
@@ -481,5 +481,5 @@ Ethernet508     508                                  etp64e     64       100000
 Ethernet509     509                                  etp64f     64       100000
 Ethernet510     510                                  etp64g     64       100000
 Ethernet511     511                                  etp64h     64       100000
-Ethernet512     512                                  etp65      65       10000
-Ethernet520     520                                  etp66      66       10000
+Ethernet512     512                                  etp65      65       25000
```

Comment:
hi Noa, these 2 ports should be 10G in the HWSKU.

Reply:
> hi Noa, these 2 ports should be 10G in the HWSKU.

Hi @r12f , We know. It was temporary solution for an issue which is no longer relevant, so this PR will be closed.